quasar/variables.tf

```diff
@@ -0,0 +1,15 @@
+/*
+ ******************************************************
+ * Copyright (C) 2019 tropos.io <team@tropos.io>
+ *
+ ******************************************************
+ */
+
+variable "fivetran_cloudwatch_integration_external_id" {
+  description = "External ID for the trust relationship between fivetran and AWS"
+}
+
+variable "aws_region" {
+  description = "The region where the resources will be deployed"
+  default = "eu-west-1"
+}
```

Comment:
This variable seems to be unused. (We default to `us-east-1` in our provider [here](https://github.com/DoSomething/infrastructure/blob/5d59c6309a0cd661c5a0933838f970c6ba08bc0d/quasar/main.tf#L13)).